packages/modules/create-imodel/src/components/button-bar/ButtonBar.tsx

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import "./ButtonBar.scss";
+
+import { Button } from "@itwin/itwinui-react";
+import React from "react";
+
+import { IModelContext } from "../context/imodel-context";
+
+export type ButtonBarProps = {
+  /** Confirm action callback function */
+  confirmAction?: (input: () => void) => void;
+  /** Is confirm button disabled */
+  isConfirmDisabled?: boolean;
+  /** Is cancel button disabled */
+  isCancelDisabled?: boolean;
```

Comment:
when/why cancel button would be disabled?

Reply:
there was a scenario we were thinking of disabling it when file uploading in-progress
right now are not passing this prop as we will be showing overlay while file upload in-progress
should we remove this prop then?

Reply:
well yes, why have something which is not needed